### DISCUSSION TITLE
emcee sampler error "requiv_max constraint"

### DISCUSSION THREAD

**Discussion by danangsetyo5364** [phoebe-project/phoebe2#860] [Questions]
Hallo! I am going to run emcee sampler  with light curve and RV data for detached binary. Here are my initial parameters distribution, and the priors. 

![gambar](https://github.com/phoebe-project/phoebe2/assets/108653905/cfa03f96-6c6a-4528-b5e7-5922ebf18abc)
![gambar](https://github.com/phoebe-project/phoebe2/assets/108653905/377773d6-e7ae-4766-88c0-031862e5937b)

I am using PHOEBE backends, but I wonder why my run is always showing same error message : `Constraint 'requiv_max@primary@star@constraint' raised the following error while attempting to solve for 'requiv_max@primary@star@component'`.

I have flip the necessary contraints, I dont know why its happens. Any idea? Thanks!

![gambar](https://github.com/phoebe-project/phoebe2/assets/108653905/3537b24c-ff93-4606-a937-abe3087f93da)

**Comment by kecnry** [comment]
It looks like the constraint had previously failed to run because of a `nan` in one of the inputs.  Can you please paste the output of `print(b.get_parameter('requiv_max', component='primary', context='component').constrained_by)`?
